roles/core/nic/readme.rst

```diff
@@ -51,7 +51,13 @@ To configure an LACP bonding, specify slave interfaces, and then create the bond
       type: bond-slave
       master: bond0
 
-To configure a vlan, simply set vlan to true:
+To configure a VLAN, simply set VLAN to true.
```

Comment:
The parameter is `vlan` (lowercase): ... set vlan to true.

Reply:
good
thank you